src/js/steps/initMonkey.js

```diff
@@ -21,6 +23,26 @@ module.exports = function ($events, options) {
 
     data.monkeyContainer.addClass('ready');
 
+    // Track errors on image load
+    var $images = data.monkeyContainer.find('[class*="page"] img');
+    $images.on('error', function () {
+      if (window.analytics) {
+        analytics.track('Broken image in monkey', { src: this.src });
```

Comment:
Would it be possible to add `pageID` to this as well (like `src`)? Would make it easier to filter.